graph/protocol.go

```diff
@@ -275,19 +273,3 @@ func addToMetadataResponses(md map[string]interface{}, k, code, flags string, v
 		md[k] = Responses{code: {flags: v}}
 	}
 }
-
-func averageMetadataValue(md map[string]interface{}, k string, v float64) {
```

Comment:
Please just comment this one out with comment:
// averageMetadataValue is currently unused but shows how to perform averaging using metadata values.

Reply:
Done!